common/Services/ComputeSystemService.cs

```diff
@@ -12,11 +12,25 @@
 using DevHome.Common.Models;
 using DevHome.Logging;
 using Microsoft.Windows.DevHome.SDK;
+using Windows.ApplicationModel;
 
 namespace DevHome.Common.Services;
 
 public class ComputeSystemService : IComputeSystemService
 {
+    private const string DevHomePreviewPackageFamilyName = "Microsoft.Windows.DevHome_8wekyb3d8bbwe";
+
+    private const string DevHomeDevPackageFamilyName = "Microsoft.Windows.DevHome.Dev_8wekyb3d8bbwe";
+
+    private const string DevHomeCanaryPackageFamilyName = "Microsoft.Windows.DevHome.Canary_8wekyb3d8bbwe";
+
+    private readonly HashSet<string> _devHomePackageFamilyName = new()

```

Comment:
For such a small size, a list would be better
https://stackoverflow.com/questions/150750/hashset-vs-list-performance

Reply:
So, I looked at the link but I think they are testing something different there. In the accepted answer, the user was creating the HashSet then adding items to it every iteration. Same with the List, however here, we create the HashSet at class construction time with items inside it one time, then we use the HashSet to check if specific items are contained in it. The speed issue only happens if we're always adding and removing items in the HashSet. Here we only create it once, since the ComputeSystemService is a singleton. HashSet<>. Also I don't see the accepted answer doing any comparisons only adding and removing. Contains is still fast in this case and the HashSet will live for the lifetime of the app and I won't need to worry about adding/removing items to it, or needing random access look up either. The only overhead it has is at class creation time, and since we're using a Host container that is constructed at app creation time, the performance hit of that is negligible. I'm thinking I rather have speed up when looking up  times in the for loop, so I'll leave this as is for now.